### PR TITLE
916 - Clean up the portal management command test

### LINKED ISSUE #916: Clean up the portal management command test
### Context

Feature branch: `feature/loader`

In PR #909, we introduced `test_readme_file` which asserts the readme contents for all computed file types defined in the `common`'s download configs. 

However, the obsolete assertion for the readme content are still present in `test_create_portal_metadata` and needs to be cleaned up.

### Problem or idea

We should remove the obsolete readme assertion from `test_create_portal_metadata`. 

Additionally, we'll replace the current hard-coded value for the download config with the constant defined in `common` as part of this issue.

### Solution or next step

1. Remove the `assertProjectReadmeContent` method and related code block
2. Replace the hard-coded download config value with `common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG`

### PR DESCRIPTION
**Pull request by nozomione**
## Issue Number

Closes #916

## Purpose/Implementation Notes
I've cleaned up `assertProjectReadmeContent` and replaced the hard-coded download config value with `common.GENERATED_PORTAL_METADATA_DOWNLOAD_CONFIG`.

## Types of changes
- Refactor (addresses code organization and design mentioned in corresponding issue)

## Functional tests
N/A

## Checklist

- [x] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added necessary documentation (if appropriate)
- [ ] Any dependent changes have been merged and published in downstream modules

## Screenshots
N/A

### CODE REVIEW

File: api/scpca_portal/test/management/commands/test_create_portal_metadata.py [hunk 1]
```diff
@@ -109,14 +78,9 @@ def test_create_portal_metadata(self, mock_upload_output_file):
...
-        download_config = {
-            "modality": None,
-            "format": None,
-            "includes_merged": False,
-            "metadata_only": True,
-            "portal_metadata_only": True,
-        }
+        # Make sure all fields match the download configuration (excludes excludes_multiplexed)
+        download_config = common.PORTAL_METADATA_DOWNLOAD_CONFIG.copy()
+        download_config.pop("excludes_multiplexed", None)
```
Comment (davidsmejia) [inline api/scpca_portal/test/management/commands/test_create_portal_metadata.py:83]: lets replace this with assertions for expected values in the same way we handle `file_size_in_bytes`

**Review by davidsmejia — COMMENTED** [review]
Just one comment otherwise looks good

File: api/scpca_portal/test/management/commands/test_create_portal_metadata.py (hunk 1 above)
Comment (nozomione) [reply api/scpca_portal/test/management/commands/test_create_portal_metadata.py]: I've added the individual assertions for the fields defined in the portal metadata `computed_file` in 938cd6a0175d8ef605161a918572bed7cad78380. This PR is ready for another look. Thank you David!

**Review by davidsmejia — APPROVED** [review]
LGTM

**Pull request merged**